string model
---
t2_package/robot_id robotID
